### DISCUSSION TITLE
[Question] CVSS Base score

### DISCUSSION THREAD

**Discussion by pkonnov** [quay/clair#1247] [Q&A]
### Description of Problem / Feature Request

There is a table with vulnerabilities(vuln) in the database. This table has a field vulnerable_range. What should it contain? 

### Expected Outcome

BaseScore CSVV3.0
BaseScore CSVV2.0
Some metadata

### Actual Outcome

'empty'

### Environment

- Clair version/image: https://github.com/quay/clair/blob/main/docker-compose.yaml

**Comment by hdonnay** [comment]
That's part of the internals for version matching.

It's not relevant to any CVSS scoring.
